libs/accounts/src/lib/accounts-data-provider.ts

```diff
@@ -55,8 +55,8 @@ export const getId = (
 const update = (
   data: Accounts_party_accounts[],
   delta: AccountSubscribe_accounts
-) =>
-  produce(data, (draft) => {
+) => {
```

Comment:
chore (non-blocking): same here

Reply:
This was requested by Matt, he prefer `"arrow-body-style": ["error", "always"]`